kythe/typescript/README.md

```diff
@@ -1,5 +1,241 @@
 # Kythe indexer for TypeScript
 
+## VName Specification
+
+This specification defines how the indexer expresses VNames for TypeScript
+declarations. You may find this useful if you are developing an application that
+relies on TypeScript code you don't want to re-index.
+
+[spec tests](./testdata/declaration_spec.ts)
```

Comment:
I think this doesn't render in the output (?)

Reply:
It does on GitHub